### ISSUE TITLE
Warning messages while running shadowcljs release app

### ISSUE THREAD

**Issue by aphsa** [reagent-project/reagent#523]
Not sure whether this is correct place to post this:
Are these messages normal for reagent 1.0.0

```
shadow-cljs release app
shadow-cljs - config: /home/jenkins/code/clojure/myproject/shadow-cljs.edn
[:app] Compiling ...
------ WARNING #1 -  -----------------------------------------------------------
 Resource: com/cognitect/transit.js:649:8
 variable module is undeclared
--------------------------------------------------------------------------------
------ WARNING #2 -  -----------------------------------------------------------
 File: ~/.m2/repository/reagent/reagent/1.0.0/reagent-1.0.0.jar!/reagent/core.cljs:370
--------------------------------------------------------------------------------
 367 |                                 tmpl/default-compiler*
 368 |                                 compiler)))
 369 |
 370 | (defn render
--------------------------------------------------------------------------------
 unreachable code
--------------------------------------------------------------------------------
 371 |   {:deprecated "0.10.0"}
 372 |   [& _]
 373 |   (throw (js/Error. "Reagent.core/render function was moved to reagent.dom namespace in Reagent v1.0."))
 374 |   nil)
--------------------------------------------------------------------------------
nil
[:app] Build completed. (141 files, 1 compiled, 0 warnings, 11.46s)
```

**Comment by Deraen** [comment]
First one is not related to Reagent.

Not sure about second one, I don't know what it is saying is unreachable code, looks fine to me.

**Comment by mike-thompson-day8** [comment]
@Deraen 
Regarding the second one:  the `throw` means the `nil` return below it will never be reached.  Solution: remove the `nil` return?

**Comment by Deraen** [comment]
Fixed in edd3c59c86c921bc5bf92abcf86f7fa215dde11e
